python-client/giskard/push/perturbation.py

```diff
@@ -195,23 +190,24 @@ def _numeric(
     transformation_functions_params,
     value_perturbed,
 ):
-    # Compute the MAD of the column
-    mad = compute_mad(ds.df[feature])  # Small issue: distribution might not be normal
+    # Compute 10% around the value to be perturbed
+    value_to_perturb = ds_slice.df[feature].iloc[0]
```

Comment:
previously we were calculating MAD based on the whole series, now we're taking just the first element, isn't it less robust in case the first element is an outlier?

Reply:
by first element, it's the sample we get in the debugger. Yes, @jmsquare suggested based on @AbSsEnT 's feedback, that we reduce the perturbation to a fixed rate (in this case 10% or less of the sample-value) instead of using MAD because it's less clear where it comes from, plus the perturbation was really sizeable (see the linear card for examples).